pcsx2/GS/Renderers/DX11/GSDevice11.cpp

```diff
@@ -341,7 +342,8 @@ bool GSDevice11::Create()
 			return false;
 	}
 
-	CreateCASShaders();
+	CreateCASShaders(support_feature_level_11_0);
```

Comment:
Probably cleaner to just do
```
if (support_feature_level_11_0)
  CreateCASShaders();
else
  m_features.cas_sharpening = false;
```

Reply:
I did thought about it first but then changed my mind so it matches gl more and all the checks are in one place this way.

Reply:
Tbh I'd suggest changing GL then as well, I wrote it that way before I considered GPUs that wouldn't support it.

Reply:
That should do it.